bin/get-json-config.py

```diff
@@ -79,11 +79,18 @@ def json_to_stream(json_str):
         return None
     for key in json_obj:
         val = json_obj[key]
-        stream += key
-        # keys without values do not add ":" e.g. --endpoint k8s,
-        if len(val) > 0:
-            stream += ":" + val
-        stream += ","
+        # converge to key:[list] if key:str to have a common structure
+        if isinstance(val, str):
+            val = [val]
+        elif not isinstance(val, list):
+            raise Exception("Error: Unexpected object type %s" % (type(val)))
+
+        for idx in range(len(val)):
```

Comment:
Also possible to use `enumerate` here. just trying to spread some options around. the `range(len(val))` is perfectly acceptable.

Reply:
True, enumerate favors readable code, but range+len is a little faster. For small loops, it does not make any difference, but I usually choose the more primitive constructions... 
https://stackoverflow.com/questions/11990105/rangelenlist-or-enumeratelist